server/plugin_test.go

```diff
@@ -148,6 +157,15 @@ func TestSpecialCases(t *testing.T) {
 		}, {
 			"![  Mattermost  ][1]\n\n[1]: https://mattermost.com/example.png",
 			"![  Mattermost  ][1]\n\n[1]: https://mattermost.com/example.png",
+		}, {
+			"Why not visit https://mattermost.com?",
```

Comment:
I know these are just unit tests, but what happens for:
```
Why not visit https://mattermost.com?important_query_string_parameter
```

Reply:
This is a matter of your regular expression. In this case, this test simply do not capture any query param, but is up to you to write the proper regular expresion.